seed/data_importer/match.py

```diff
@@ -72,14 +125,17 @@ def match_and_link_incoming_properties_and_taxlots(file_pk, progress_key, sub_pr
     returned as a dict.
 
     :param file_pk: ImportFile Primary Key
+    :param cycle: cycle object
     :return results: dict
     """
+
     from seed.data_importer.tasks import pair_new_states
 
     import_file = ImportFile.objects.get(pk=file_pk)
-    progress_data = ProgressData.from_key(progress_key)
     update_sub_progress_total(100, sub_progress_key)
 
+    progress_data = ProgressData.from_key(progress_key)
+
```

Comment:
I wasn't really sure how to handle the `progress_data` related pieces of the code. I ended up moving this and the associated other lines of code into the `_by_cycle` method because I was having a hard time figuring out how to pass the `merged_linked_property_views` and `merged_linked_taxlot_views` back to the main method.

Reply:
I mean just this line moving from L80 (now L131) to here?